Author: Jamie Strandboge <jamie@ubuntu.com>
Description: adjust qemu.conf to run as root, not the new Debian default of
 libvirt-qemu:kvm
 .
 In Ubuntu, Qemu/KVM virtual machines are fully isolated and confined by the
 AppArmor security driver. This new default in Debian is not as well-tested,
 with the upstream implementation contentious among the KVM and libvirt
 communities.
 .
 While the current implementation does reduce the privileges of the VMs
 running under qemu:///system, all VMs currently run under the same user so
 there is no guest isolation. Even if each user ran under its own user, an
 attacker could potentially break out of the VM and have unconfined user
 access (albeit non-root).
 .
 Once there is consensus among the community on the implementation, changing
 this default in Ubuntu can be considered as an additional protection to the
 AppArmor driver.

diff -Nur libvirt-0.7.5/src/qemu/qemu.conf libvirt-0.7.5.new/src/qemu/qemu.conf
--- libvirt-0.7.5/src/qemu/qemu.conf	2009-12-22 03:37:57.000000000 -0600
+++ libvirt-0.7.5.new/src/qemu/qemu.conf	2010-01-22 17:21:16.189521666 -0600
@@ -91,10 +91,12 @@
 
 
 # The user ID for QEMU processes run by the system instance
-#user = "root"
+#user = "libvirt-qemu"
+user = "root"
 
 # The group ID for QEMU processes run by the system instance
-#group = "root"
+#group = "kvm"
+group = "root"
 
 
 # What cgroup controllers to make use of with QEMU guests
